src/stories/Tooltip.stories.tsx

```diff
@@ -22,12 +22,12 @@ const Template: Story<ComponentProps<typeof Tooltip>> = args => {
   const [show, setShow] = React.useState<boolean>(false);
   React.useEffect(() => {
     if (ref.current) setShow(true);
-  }, [ref.current]);
+  }, []);
```

Comment:
why remove the check? if `ref.current` is always true then should we remove the if condition as well?

Reply:
eslint told me not to put ref.current in the dep array since it's not reactive?